repository/create_todo_repository.go

```diff
@@ -18,11 +19,9 @@ func (repo *newRepository) CreateTodoRepository(db *gorm.DB, todo domain.CreateT
 		CreatedAt:   todo.CreatedAt,
 	}
 
-	if err = db.Create(input).Error; err != nil {
-		err = errors.New("RepositoryError")
-		log.Println("ERR >>>>", err.Error())
+	if err = db.WithContext(ctx).Create(input).Error; err != nil {
 
-		return resp, err
+		return resp, error_lib.WrapError("repository", err)
```

Comment:
“repository” should be ‘const’ in common directory like 
const (
ErrDBCreateTodo = errors.New(“failed to create todo repository”)
)